tests/cli/test_cli.py

```diff
@@ -719,6 +719,13 @@ def test_upload1(ctx):
         )
         upload_suffix.append(['--store', bundle_store_name])
 
+    if os.environ.get("CODALAB_GOOGLE_APPLICATION_CREDENTIALS") is not None:
```

Comment:
No, we do not run automatic tests for GCS (since we don't want to push our credentials to github). Instead, I run the tests locally, by setting `CODALAB_GOOGLE_APPLICATION_CREDENTIALS`. 

I will update our manual test documents about this part.

Reply:
I tested bypass server locally (since we don't want to exposure our google credential to github). And it looks good and paased all the tests.
<img width="1349" alt="image" src="https://user-images.githubusercontent.com/40016222/193198315-185cb77a-e3ab-4520-a6ad-91199fb269ee.png">

It follows test GCS steps in `multiple bundle storage` document.